source/common/http/conn_pool_grid.cc

```diff
@@ -83,6 +89,17 @@ void ConnectivityGrid::WrapperCallbacks::deleteThis() {
 }
 
 bool ConnectivityGrid::WrapperCallbacks::newStream() {
+  if (grid_.is_http3_broken()) {
```

Comment:
Any reason to not create the wrapper callback pointing at the H2 pool if H3 is broken?

Reply:
Sure, that works. Done.